slang-gfx.h

```diff
@@ -1313,16 +1318,27 @@ struct RayTracingPipelineStateDesc
 class IShaderTable : public ISlangUnknown
 {
 public:
+    // Specifies the bytes to overwrite into a record in the shader table.
+    struct ShaderRecordOverwrite
+    {
+        uint32_t offset; // Offset within the shader record.
+        uint32_t size; // Number of bytes to overwrite.
+        uint8_t data[8]; // Content to overwrite.
+    };
```

Comment:
I was going to ask if this should be "override" instead of "overwrite," but as I look at the details it sure seems like it really is about overwriting specific bytes.

I can imagine future extensions to this might need to:

*  make `data` a pointer rather than a fixed-size array
* add a parallel `mask` array/pointer to allow the overwrite to touch only specific bits in the shader record

I think it would be premature to add those generalizations right now, though.